src/python/py/models/README.md

```diff
@@ -4,24 +4,27 @@ This folder contains the model builder for quickly creating optimized and quanti
 
 # Contents
 
-- [Current Support](#current-support)
-- [Usage](#usage)
-  - [Full Usage](#full-usage)
-  - [Original PyTorch Model from Hugging Face](#original-pytorch-model-from-hugging-face)
-  - [Original PyTorch Model from Disk](#original-pytorch-model-from-disk)
-  - [Customized or Finetuned PyTorch Model](#customized-or-finetuned-pytorch-model)
-  - [Quantized PyTorch Model](#quantized-pytorch-model)
-  - [GGUF Model](#gguf-model)
-  - [Extra Options](#extra-options)
-    - [Config Only](#config-only)
-    - [Exclude Embedding Layer](#exclude-embedding-layer)
-    - [Exclude Language Modeling Head](#exclude-language-modeling-head)
-    - [Enable Cuda Graph](#enable-cuda-graph)
-    - [Use 8 Bits Quantization in QMoE](#use-8-bits-quantization-in-qmoe)
```

Comment:
Can we just keep the insertion of the new "Hugging Face Authentication" section below "Use 8 Bits Quantization in QMoE" and then keep the original table of contents formatting as is? I think the first two links, "ONNX Runtime GenAI Model Builder" and "Contents", are self-explanatory since a user can see the README title and the table of contents heading as soon as the README doc is opened.

My concern is that users may click on the new "ONNX Runtime GenAI Model Builder" and "Contents" links, not see the page move since the user is already at those positions in the README, and then wonder why the links are not working (even though it is).

Reply:
Very sorry for this, my VSCode Markdown Extension automatically adjusted the ToC.... this was not intended. I think this is now how it should be or ?

Reply:
Looks good to me now. Thank you!